tests/extensions/FlakyTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Appwrite\Tests;
+
+/**
+ * Allows test methods to be retried if they fail.
+ *
+ * Requires that the test class extends {@see TestCase} and has trait {@see Retryable}.
+ */
+#[\Attribute(\Attribute::TARGET_METHOD)]
+class FlakyTest
```

Comment:
I think its better to call this `Retry` with the constructor parameter as `$count` . That way we would call it as

`#[Retry(count: 1)]`